README.md

```diff
@@ -5,39 +5,363 @@
+
+![Subscriptions Issues Link](https://github.com/DataDog/gello/blob/master/images/subscriptions_highlight_pull_requests.png)
+
+
+## How It Works
+
+_Gello_ works by _subscribing_ Trello boards to GitHub repositories through a web UI, and selecting the corresponding Trello lists you would like cards to be created to following a GitHub event.
+
+### GitHub Webhooks
+_Gello_ uses [GitHub webhooks](https://developer.github.com/webhooks/) to get event updates from the repositories subscribed.
+
+When a subscription is made, _Gello_ will create a GitHub webhook on the corresponding repository with the following permissions: `Issues`, `Issue comments`, `Pull requests`, and `Pull request review comments`.
+
+This will allow the server to receive webhook events whenever an issue or pull request is created or commented upon. They are necessary for both the _autocard_ and _manual_ settings.
+
+### GitHub Events
+#### Pull Request Event
+
+For the _autocard_ setting, if a pull request is created by a person outside of the members of a GitHub organization on a subscribing repository, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Issue Event
+
+Likewise, for the _autocard_ setting, if an issue is created by a person outside of the members of a GitHub organization on a subscribed repository, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Pull Request Comment Event
+
+For the _manual_ setting, if a pull request has been created, and a member of the GitHub organization comments `gello create_card` on the pull request, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Issue Comment Event
+
+For the _manual_ setting, if an issue has been created, and a member of the GitHub organization comments `gello create_card` on the issue, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+
+## Configuration
+
+### Configure the Server
+
+_Gello_ requires certain environment variables to be set for the server to be run correctly.
+
+```bash
+# Admin user configuration
+ADMIN_EMAIL='an_email_account@gmail.com'
+ADMIN_PASSWORD='an_admin_password' # create a password for logging into gello
+
+# Database configuration
+DATABASE_URL='the_url_for_a_postgresql_database' # likely postgresql://localhost:5432/your_postgresql_database_name
+
+# Redis configuration
+REDIS_URL='the_url_for_a_redis_client' # defaults to 'redis://localhost:6379/0'
+
+# GitHub configuration values
+GITHUB_API_TOKEN='An API token for a user with access to your GitHub organization'
+GITHUB_ORG_LOGIN='The login for your GitHub organization' # also the name for your Github organization
+
+# Trello configuration values
```

Comment:
I haven't looked into this option - let me get back to you on this!

Reply:
I haven't stumbled upon a case where this was needed, but it's a great way to test out how the Trello API works! I could reference it dev guide (maybe) in case it's needed in the future :)